libcloud/container/base.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         ip_addresses,  # type: List[str]
         driver,  # type: ContainerDriver
         extra=None,  # type: dict
+        created_at=None,  # type: str
```

Comment:
Yeah, I think it should be fine adding it to the base API since it's added to the end and shouldn't break existing code.